laspy/vlrs/known.py

```diff
@@ -442,7 +442,7 @@ class GeoKeysHeaderStructs(ctypes.LittleEndianStructure):
 
     def __init__(self):
         super().__init__(
-            key_directory_version=1, key_revision=1, minor_revision=0, number_of_kets=0
+            key_direction_version=1, key_revision=1, minor_revision=0, number_of_keys=0
```

Comment:
`key_directory_version`  is the correct name, the one in `__fields__` is incorrect

Reply:
I've updated that now. Thanks for looking it over!